### ISSUE TITLE
About the project...

### ISSUE THREAD

**Issue by LedZeck** [newswim/materializeLayout#1]
Was stopped?

**Comment by newswim** [comment]
Yes, on hiatus for now. I'll trying to get more features up and running this summer.

**Comment by LedZeck** [comment]
I see...

Well... i'm very interested on materialize projects. So... If you restart this one, i can help =D
